internal/pkg/cli/app_package.go

```diff
@@ -275,38 +302,23 @@ func (o *packageAppOpts) getAppTemplates(env *archer.Environment) (*appCfnTempla
 			projAccountID: proj.AccountID,
 		}
 	}
-
-	switch t := mft.(type) {
-	case *manifest.LoadBalancedWebApp:
-		appMft := mft.(*manifest.LoadBalancedWebApp)
-		appStack, err := initLBFargateStack(appMft, env.Name, env.Project, stack.RuntimeConfig{
-			ImageRepoURL:   repoURL,
-			ImageTag:       o.Tag,
-			AdditionalTags: proj.Tags,
-		}, proj.RequiresDNSDelegation())
-		if err != nil {
-			return nil, err
-		}
-
-		tpl, err := appStack.Template()
-		if err != nil {
-			return nil, err
-		}
-		params, err := appStack.SerializedParameters()
-		if err != nil {
-			return nil, err
-		}
-		return &appCfnTemplates{stack: tpl, configuration: params}, nil
-	default:
-		return nil, fmt.Errorf("create CloudFormation template for manifest of type %T", t)
+	serializer, err := o.stackSerializer(mft, env, proj, stack.RuntimeConfig{
+		ImageRepoURL:   repoURL,
+		ImageTag:       o.Tag,
+		AdditionalTags: proj.Tags,
+	})
+	if err != nil {
+		return nil, err
 	}
-}
-
-var initLBFargateStack = func(app *manifest.LoadBalancedWebApp, env, proj string, rc stack.RuntimeConfig, isHTTPS bool) (stackSerializer, error) {
-	if isHTTPS {
-		return stack.NewHTTPSLoadBalancedWebApp(app, env, proj, rc)
+	tpl, err := serializer.Template()
```

Comment:
so this tpl refers to Cloudformation Template, not html/Template, correct?

Reply:
Correct, it refers to a template provided by the `deploy/stack` pkg, like:
https://github.com/aws/amazon-ecs-cli-v2/blob/e1cf3445c3923a5364dab0a1dc27b7e204c33a31/internal/pkg/deploy/cloudformation/stack/lb_web_app.go#L82